testdata/scripts/health/multi-chain-loopp.txtar

```diff
@@ -71,7 +78,25 @@ ChainID = 'Baz'
 Name = 'primary'
 URL = 'http://stark.node'
 
+[[Ton]]
+ChainID = '99'
+
+[[Ton.Nodes]]
+Name = 'node'
+URL = 'https://ton.rpc'
+
+[[Tron]]
+ChainID = '1001'
+
+[[Tron.Nodes]]
+Name = 'example'
+URL = 'http://tron.org'
+SolidityURL = 'https://solidity.evm'
+
 -- out.txt --
+ok Aptos.42.RelayerService
+ok Aptos.42.RelayerService.PluginRelayerClient
+ok Aptos.42.RelayerService.PluginRelayerClient.PluginAptos
```

Comment:
Looks like aptos is either not starting cleanly, or the health report is not wired through... :thinking: